yarn-project/noir-compiler/src/__snapshots__/index.test.ts.snap

```diff
@@ -3,9 +3,167 @@
 exports[`noir-compiler using nargo binary compiles the test contract 1`] = `
 [
   {
+    "debug": {
```

Comment:
@spalladino was refering to here

Reply:
My guess is it may change whenever we bump the noir version we're using